### DISCUSSION TITLE
Error icon adjustment on help text in labels

### DISCUSSION THREAD

**Discussion by chjpark** [twilio-labs/paste#1062] [Q&A] [answered]
Does the helptext have anything where you can adjust the position of the error icon? 

For instance:
<img width="318" alt="Screen Shot 2021-01-04 at 9 44 25 AM" src="https://user-images.githubusercontent.com/1145733/103563322-7409c380-4e71-11eb-837e-c15e13682ad9.png">

I'd like the error icon above to be aligned with the first line of the text.

**Comment by corinne-nagel** [comment]
Hi, @chjpark , I think @richbachman can answer the positioning question, but I also think this help text could be edited down to just one line:

🛑  Please [add a phone number](url) to this service

We suggest always keeping help/error text as concise as possible. Hope this helps!

**Comment by richbachman** [comment, marked as answer]
Hi @chjpark. That icon should be top aligned. It's a bug on our end. We'll get that fixed and let you know when the new version is published.

**Reply by richbachman** [reply]
Hi @chjpark, we just merged in a fix for this. Once everything is deployed you should just have to update your Paste dependencies. Let us know if you have any other issues.
